### DISCUSSION TITLE
Data Source from WebAPI

### DISCUSSION THREAD

**Discussion by alberk8** [ariacom/Seal-Report#89] [Q&A] [answered]
Is it possible to read data from WebAPI that returns JSON data and generate a report off that?

**Comment by ariacom** [comment]
No, but it could be implemented by generating a report in JSON, thus report models will generate a Json data based on its Result Pages and Tables...
It would be a new report format.

**Reply by alberk8** [reply]
I have a data source that is accessible via web api (json format),   Can I connect to that "data source" and create a report?

**Reply by ariacom** [reply, marked as answer]
You should check: https://sealreport.org/Sources#lineNoSQL
And check the sample: 02-LINQ\201-LINQ Samples CSV-Excel-JSON-XML
